src/color.scss

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2024 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Import Auro tokens
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
+
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/src/breakpoints';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/src/core';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/src/utilityClasses/displayProperties';
+
+/* stylelint-disable max-nesting-depth, selector-class-pattern */
+
+:host {
+  [auro-dropdown] {
+    &::part(trigger) {
+      &:hover {
+        --ds-auro-dropdown-trigger-container-color: rgba(0 0 0 / 0.06); // Needs container token
+      }
+    }
+  }
+
+  .selectElement-helpText {
+    color: var(--ds-auro-select-help-text-color);
+  }
+}
+
+:host([disabled]) {
+  * {
+    color: var(--ds-color-text-disabled-default, $ds-color-text-disabled-default);
+  }
+}
+
+:host([validity]:not([validity='valid'])) {
+  .selectElement-helpText {
+    --ds-auro-select-help-text-color: var(--ds-color-text-error-default, #{$ds-color-text-error-default});
```

Comment:
Why does select have to do any work on the help text coloring at all? This should be handled by dropdown entirely.

Reply:
That is a great point that we can discuss. Looking at the javascript code, it looks like auro-select is currently handling its own validation, which should be handled at the dropdown level.

Reply:
dropdown doesn't "do" validation. But it does render the results of the validation simply by adding/removing the "error" attribute to the dropdown element in the render function.

I would think you should be able to just remove all the select style code for validation and, verify the error attribute is passed to dropdown correctly, and then retest it.